homeassistant/components/automation/__init__.py

```diff
@@ -460,7 +460,11 @@ async def async_trigger(self, run_variables, context=None, skip_condition=False)
         ) as automation_trace:
             if self._variables:
                 try:
-                    variables = self._variables.async_render(self.hass, run_variables)
+                    states = self.hass.states.get(self.entity_id)
```

Comment:
Name the variable `state` since it's a single state

Reply:
Done